packages/jest-jasmine2/src/index.js

```diff
@@ -69,7 +69,49 @@ function jasmine2(config, environment, moduleLoader, testPath) {
       moduleLoader.requireModule(null, config.setupTestFrameworkScriptFile);
     }
   });
+
+  const hasIterator = object => !!(object !== null && object[Symbol.iterator]);
+  const jestCustomEqualityTesters = [];
```

Comment:
because we only have one of these, can we just call `jasmine.addCustomEqualityTester(tester)` directly? We call this function a lot, so I don't really want to add this array overhead unless we need it.